android/java/org/chromium/chrome/browser/rate/BraveAskPlayStoreRatingDialog.java

```diff
@@ -65,6 +67,11 @@ public void show(@NonNull FragmentManager manager, @Nullable String tag) {
     @Override
     public void setupDialog(@NonNull Dialog dialog, int style) {
         super.setupDialog(dialog, style);
+        mReviewManager = ReviewManagerFactory.create(getActivity());
+        try {
+            requestReviewFlow();
+        } catch (NullPointerException e) {
+        }
```

Comment:
i think it's worth noting this error. we can add log here.

Reply:
updated